playwright/integrations/authenticated-chat-with-persistantchat.spec.ts

```diff
@@ -0,0 +1,63 @@
+import fetchOmnichannelConfig from '../utils/fetchOmnichannelConfig';
+import fetchTestPageUrl from '../utils/fetchTestPageUrl';
+import fetchAuthUrl from '../utils/fetchAuthUrl';
+import { test, expect } from '@playwright/test';
+import OmnichannelEndpoints from '../utils/OmnichannelEndpoints';
+
+const testPage = fetchTestPageUrl();
+const omnichannelConfig = fetchOmnichannelConfig('AuthenticatedChatWithChatPersistent');
+const authUrl = fetchAuthUrl('AuthenticatedChatWithChatPersistent');
+
+test.describe('AuthenticatedChat @AuthenticatedChatWithChatPersistent', () => {
+    test('Persistent Chat- ChatSDK.endChat() without any reconnect id should call session close with isPersistentChat=true as query params', async ({ page }) => {
```

Comment:
It should be `ChatSDK.endChat() without any reconnect id should call session close with isPersistentChat=true as query params`